### ISSUE TITLE
error mountain_car_1

### ISSUE THREAD

**Issue by Ser271155** [sol0invictus/MAT-DL#1]
Sunny.
type conversion error occurs when training is started:
"Conversion of Python 

element at position 1 to type ‘double’ failed....
![error](https://user-images.githubusercontent.com/116013932/208929689-75c3d13b-7d9f-4e40-9664-f71a2fa1e423.png) #

**Comment by ravigandhi2019** [comment]
Yes, I tried to change different data types. But, still shows the error.
